source/_integrations/sonos.markdown

```diff
@@ -248,6 +248,39 @@ data:
   enqueue: play
 ```
 
+## Selecting Source
+
+The `media_player.select_source` service may be used to select specific sources on your Sonos.
```

Comment:
This is a service from the `media_player` platform integration and is documented there.

<https://www.home-assistant.io/integrations/media_player#service-media_playerselect_source>

We don't want to duplicate all services across every integration that provides entities for a platform.

Reply:
@frenck I'm trying to document the platform dependent parameters for Sonos.  For Sonos source can be either TV, Line-in or the name of a favorite.  I copied this from the Roku documentation and changed the Roku platform dependent parameters to the Sonos ones.